packages/constants/src/constants.ts

```diff
@@ -304,11 +304,13 @@ export const STORAGE_KEYS = {
 } as const;
 
 export const HTTP_SECURITY_HEADERS = {
+  'Referrer-Policy': 'same-origin',
+  'Permissions-Policy':
+    'microphone=(), camera=(), payment=(), usb=(), geolocation=()',
```

Comment:
This is added now, so that when parsing the headers from the config we retain the default values.
It also makes it more transparent what are the default values being used.